systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterIT.java

```diff
@@ -325,7 +325,7 @@ public void testJvmAndResources() {
                 "-Xmx1g", "-Xms1G", "-server", "-XX:+UseG1GC");
 
         assertResources(NAMESPACE, "jvm-resource-cluster-zookeeper-0",
-                "1G", "300m", "1G", "300m");
+                "1Gi", "300m", "1Gi", "300m");
```

Comment:
This changed because that's what the resources are in the classpath resource, I assume?

Reply:
@tombentley exactly!